README.md

```diff
@@ -1,8 +1,35 @@
 # Moviepedia
 List of movies their actor/actress
-
 # Screenshot 
 ![moviesList](/assets/1.PNG)
 ![movieDeatil](/assets/2.PNG)
 ![people](/assets/3.PNG)
-![peopleBio](/assets/4.PNG)
\ No newline at end of file
+![peopleBio](/assets/4.PNG)
+### Technologies Used :
+  * Angular-JS
+  * GitHub API
+  * Netlify.com (for hosting)
+### How to Run Locally :
+1. Clone the Repository
+     `git clone https://github.com/NJACKWinterOfCode/Moviepedia.git`
+2. cd into Moviepedia
+      `cd Moviepedia`
```

Comment:
Again we have to move to the folder `Moviepedia`

Reply:
Please explain about no new line at end of file

Reply:
https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file

Reply:
I do not understand what you are saying

Reply:
Have you tried to run Moviepedia?

Reply:
yes command through install angular cli and start local host

Reply:
Please explain

Reply:
You can add a video tutorial.

Reply:
Project Setup Video Tutorial

Reply:
Yes, please add that.